### ISSUE TITLE
Lookup issues with OM with encryption

### ISSUE THREAD

**Issue by ceraolo** [OpenModelica/OpenModelica#11434]
### Description
See steps

### Steps to Reproduce  
**To reproduce I must supply a zip file separately for privacy reasons**

From the zip file
1. Open TestEncrOM
2. create encripted package (should be equal to the enclosed TestEncrOM.mol)
3. start again OMEdit
4. load encripted package

Issue 1: 
Open CVT: the following error message appears:
![image](https://github.com/OpenModelica/OpenModelica/assets/7373030/2a4d6896-9167-4e8c-b6a5-3cb6bfe2eeee)

Issue 2: open DCT: the following error message appears:
![image](https://github.com/OpenModelica/OpenModelica/assets/7373030/d9b85cb3-5708-4615-86f8-0ecfe3e96c5f)

Both CVT and DCT open, check and run smoothly before encryption.

### Expected Behavior
CVT and DCT should open and run without errors as in the original mo package.

### Version and OS
OM 1.22.beta6 with encryption
Win 10

 - OpenModelica Version: [e.g. `omc --version` or `Help->About OMEdit` from OMEdit]
 - OS: [e.g. Windows 10, 64 bit]
 - Versions of used Modelica libraries if applicable

**Comment by perost** [comment]
> @perost can you help with that? If so, @ceraolo can send you the zip file.

I don't really know anything about the encryption stuff, but in the first image the missing class begins with `TestEncr` while the library seems to be called `TestEncrOM`. So it looks like either the library was renamed without updating the references in it, or there's a separate `TestEncr` library that hasn't been loaded.

**Comment by casella** [comment]
@ceraolo does this help fixing your issue?

**Comment by ceraolo** [comment]
I started from TestEcr.mo which is a small one-file package, with encrypting annotations of the kind of MS 18.9.
Then, since  for encrypting-ready OM requires multi-file packages, I duplicated into TestEncrOM which is just TestEncr, converted into multi-file package. Then encrypted TestEncrOM, using File|Export|Encrypted package.

After https://github.com/OpenModelica/OpenModelica/issues/11434#issuecomment-1777690512 I made further tests, and can confirm that **there is an issue in the code to duplicate one-file package into multi-file**, deserving a separate ticket.

Now I created TestEncrOM2 as a multifile version of TestEncr from dymola, then encrypted it with OM. The issues mentioned in this ticket are gone.

So, now I'm going to close this. I will open a new ticket on the save-as issue.
